_posts/course-materials/earth-analytics/week-8/in-class/2017-03-08-automation02-function-example-r.md

```diff
@@ -44,7 +46,28 @@ setwd("~/Documents/earth-analytics")
 
 # load spatial packages
 library(raster)
+## Loading required package: sp
```

Comment:
Ok cool it builds now after feeding R some cookies.
the word network is way uglier so i may fix that but again i'll wait till i go back to the lessons again!! they are slated for an update. They key for this PR was really the code changes to the side bar to reflect the new ontology @mbjoseph ! i'm still redoing my class :)  but these are good catches!